packages/paste-core/components/list/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.1.0",
   "category": "typography",
   "status": "production",
-  "description": "The list component is used to render text as a list of items.",
+  "description": "A List is a collection of text objects in a bulleted or numbered format.",
```

Comment:
NIT: Should we land on either `collection` or `set`? Or is this type of collection different from a collection of pills?